modules/announcements/src/App.tsx

```diff
@@ -0,0 +1,75 @@
+import "./App.css";
+
+import "common-ui/dist/index.css";
+
+import {
+  AppShell,
+  AppShellProps,
+  initializeI18n,
+  LoginPage,
+  NavMenuItemProps,
+  RouteProps,
+} from "common-ui";
+import EditAnnouncement from "./pages/EditAnnouncement";
+import CreateAnnouncement from "./pages/CreateAnnouncement";
+import ViewAnnouncements from "./pages/ViewAnnouncements";
+
+type User = {
+  name: string;
+};
+
+const AboutPage = () => <h1>About Page</h1>;
+
+initializeI18n([], `${process.env.PUBLIC_URL}/locales/{{lng}}/{{ns}}.json`);
+
+function App() {
+  const routes: RouteProps[] = [
+    {
+      path: "/",
```

Comment:
The URLs that will be seen in the browser should be
/announcements
/announcements/create
/announcements/edit/:id

Reply:
this module is not yet used in console app. the these routes are only used when app runs as standalone.
the urls are 
https://sandbox.shikshaplatform.io/modules/announcements/...

Reply:
Got it. Do update the routes anyway @Hardik-hi and also ensure appropriate routes are used when this module is imported into the parent admin ui app.

Reply:
Sure @coolbung